karspexet/ticket/templates/payment.html

```diff
@@ -38,12 +41,26 @@
       <input name="email" class="field" placeholder="" type="email" />
     </label>
   </div>
+  {% if stripe_key == "fake" %}
```

Comment:
Logic.. in.. view.. 😿 

Would you mind moving this into the view, and use a dynamic include + partials for this instead:

```python
    ...
    locals['payment_partial'] = '_fake_payment'
    ...
```

And

```jinja
{% include payment_partial %}
```
Note the lack of quotes, it's always an expression but 99% of the time it's a string literal so people tend to forget the dynamic nature of `include`.